src/styles/product-card.scss

```
@@ -0,0 +1,89 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 31px 16px 16px;
+  background-color: $basicWhite;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    height: 135px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 5px;
+    color: $mainAccent;
+  }
+
+  &__code {
+    font-size: 10px;
+    align-self: start;
+    margin-bottom: 17px;
+    color: $secondaryColour;
+  }
+
+  &__rate {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 16px;
+    font-weight: 400;
+    color: $mainAccent;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 15px;
+  }
+
+  &__price-label {
+    color: $secondaryColour;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    color: $mainAccent;
+  }
```

better variant(how u can use SCSS)
```suggestion
  &__price {
    width: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 15px;
    
    &-label {
      color: $secondaryColour;
     }

    &-value {
      font-size: 16px;
      font-weight: 700;
      color: $mainAccent;
    }
  }
```